### ISSUE TITLE
Object Detection API : SSD classification loss in Object Detection API is different from the original paper .

### ISSUE THREAD

**Issue by shamanez** [tensorflow/models#2213]
In the paper they have used a softmax cross entropy loss. But here in the model is using **sigmoid cross entropy loss** . 

Why is has changed ? What is the effect ? 

Check this configuration file . [This says it is weighted sigmoid](https://github.com/tensorflow/models/blob/master/object_detection/samples/configs/ssd_mobilenet_v1_pets.config#L110)

**Comment by irmowan** [comment]
You can change it to 'weighted_softmax', refer to https://github.com/tensorflow/models/blob/master/object_detection/protos/losses.proto

**Comment by shamanez** [comment]
@irmowan  yeah ! But um working with a pre trained model I think it will affect a lot to output results .

**Comment by skye** [comment]
@jch1 @tombstone @derekjchow @jesu9 @dreamdragon

**Comment by giulio-zhou** [comment]
Any updates on this? A brief comment on any empirical differences (or lack thereof) in training or general behavior would be super helpful.

**Comment by guillemdelgado** [comment]
@giulio-zhou I've trained the MobilenetV1 with SSD with the same parameters using sigmoid cross entropy and softmax cross entropy. The classification loss was different and the location loss remained the same on both experiments. However, the AP@0.5IoU decreased greatly using Softmax loss. What I could observe epirically on the evaluation dataset is that there are much more false positives.

**Comment by shishichang** [comment]
@irmowan  i change it to 'weighted_softmax',  that results in much more false positives. is there any other tricks?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
